crowbar_framework/app/models/node_object.rb

```diff
@@ -563,7 +563,12 @@ def asset_tag
     if virtual?
       "vm-#{mac.gsub(':',"-")}"
     else
-      @node[:dmi]["chassis"]["serial_number"] rescue nil
+      asset = @node[:dmi]["chassis"]["serial_number"] rescue nil
+      if @node[:dmi]["chassis"].has_key?("asset_tag") && !@node[:dmi]["chassis"]["asset_tag"].empty?
+        "#{asset} (#{@node[:dmi]["chassis"]["asset_tag"]})"
+      else
+        asset
+      end
```

Comment:
@bkutil my comments are related to @vuntz comment. And we can not split this informations, these are values provided by ohai.